packages/headless/src/features/folding/folding-actions.ts

```diff
@@ -0,0 +1,25 @@
+import {NumberValue, SchemaDefinition, StringValue} from '@coveo/bueno';
+import {createAction} from '@reduxjs/toolkit';
+import {validatePayload} from '../../utils/validate-payload';
+
+export interface RegisterFoldingPayload {
```

Comment:
For consistency, let's end the interface name with `...ActionCreatorPayload`.
Let's also document the fields on this interface, since that is what will show up on the site,